root-packages/zmap/build.sh

```diff
@@ -0,0 +1,12 @@
+TERMUX_PKG_HOMEPAGE=https://zmap.io/
+TERMUX_PKG_DESCRIPTION="a fast single packet network scanner designed for Internet-wide network surveys"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="3.0.0-beta1"
+TERMUX_PKG_SRCURL=https://github.com/zmap/zmap/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=c97c452930ad6af95ad9ad79e66fea742f904b8e102c185f1001822cc5b5ceb3
+TERMUX_PKG_DEPENDS="libgmp, json-c, libpcap"
+TERMUX_PKG_BUILD_DEPENDS="libgmp, gengetopt, libpcap, flex, byacc, json-c, pkg-config, libunistring"
```

Comment:
These build dependencies look weird, libraries are normally runtime dependencies, and tools like flex, byacc, json-c etc should be installed through [setup-ubuntu.sh](https://github.com/termux/termux-packages/blob/master/scripts/setup-ubuntu.sh) / [setup-termux.sh](https://github.com/termux/termux-packages/blob/master/scripts/setup-termux.sh)

Probably libraries should be moved to `TERMUX_PKG_DEPENDS`

Reply:
Looks like `json-c` didn't get installed on preparation. I would added it back later.